django_material_demo/polls/static/material/js/materialize.components.js

```diff
@@ -223,7 +223,11 @@ var DataTable = /*#__PURE__*/function (_HTMLElement) {
           data['datatable-' + key] = value;
         }
 
-        Object.assign(data, Object.fromEntries(new URLSearchParams(window.location.search).entries()));
+        var newData = new URLSearchParams(window.location.search);
+        for (var [k, v] of new URLSearchParams($.param(data))) {
+          newData.append(k, v);
+        }
+        return newData.toString();
```

Comment:
Is this part updating the django-material library code? Asking this because we probably want to avoid changing the library code, which means if other projects want to use it, they will also need this update.

Reply:
This is indeed a patch of the library code. To address this question, we should look for ways to add patches using additional code files, so that modifying this file is not needed